client/state/signup/steps/site-goals/actions.js

```diff
@@ -1,10 +0,0 @@
-import { SIGNUP_STEPS_SITE_GOALS_SET } from 'calypso/state/action-types';
```

Comment:
We can also remove the `SIGNUP_STEPS_SITE_GOALS_SET` action type as it's no longer used.